server.js

```diff
@@ -42,69 +54,96 @@ app.set('port', process.env.PORT || 5000);
+app.use('/favicon.png', express.static(relativePath("favicon.png")));
+
 
-// Authentication (authn)
-//
+/* Authentication (authn)
+ */
 authn.setup(app);
 
 
-/* simple logging for debugging */
-// app.use((req, res, next) => {console.log("LOG REQUEST  ", req.originalUrl); next();});
+/* Redirects.
+ */
+app.route("/auspice")
+  .get((req, res) => res.redirect('https://nextstrain.github.io/auspice/'));
 
-/* handle redirects to outside sites */
-app.get("/auspice*", (req, res) => {
-  res.redirect('https://nextstrain.github.io/auspice/');
-});
 
-/* A portion of nextstrain.org is a "static" website build with gatsby
- * This is ./static-site, with it's own package.json etc
- * ./static-site/public contains the built files we need serve
+/* Static assets.  Auspice hardcodes /dist/… in its Webpack config.
  */
-app.use(express.static(path.join(__dirname, "static-site", "public")));
-const gatsbyRoutes = [
-  "/",
-  "/about*",
-  "/blog*",
-  "/docs*",
-  "/reports*",
-  "/tutorial*",
-  "/community",
-  "/local*" // local shouldn't be handled by auspice for nextstrain.org
-];
-app.get(gatsbyRoutes, (req, res) => {
-  utils.verbose(`Sending ${req.originalUrl} to gatsby as it matches a (hardcoded) gatsby route`);
-  res.sendFile(path.join(__dirname, "static-site", "public", "index.html"));
-});
+app.use(express.static(gatsbyAssetPath()));
+
+app.route("/dist/*")
+  .all(expressStaticGzip(auspiceAssetPath()));
 
 
-/* We use auspice to display phylogenomic data.
- * Our version of auspice is based off of https://github.com/nextstrain/auspice
- * but is built with custom extensions -- see "./src/auspiceConfig.json"
- * The built code is available in the "./auspice" directory.
- * Note that if you are developing auspice in the context of nextstrain.org, it's
- * easier to do this by using the auspice dev server. See docs.
+/* Charon API used by Auspice.
  */
+app.route("/charon/getAvailable")
+  .get(auspiceServerHandlers.getAvailable);
 
-/* Handle the auspice-derived requests ("charon") */
-app.get("/charon/getAvailable", auspiceServerHandlers.getAvailable);
-app.get("/charon/getDataset", auspiceServerHandlers.getDataset);
-app.get("/charon/getNarrative", auspiceServerHandlers.getNarrative);
-app.get("/charon*", (req, res) => {
-  res.statusMessage = `Query unhandled: ${req.originalUrl}`;
-  utils.warn(res.statusMessage);
-  return res.status(500).end();
-});
+app.route("/charon/getDataset")
+  .get(auspiceServerHandlers.getDataset);
+
+app.route("/charon/getNarrative")
+  .get(auspiceServerHandlers.getNarrative);
+
+app.route("/charon/*")
+  .all((req, res) => res.status(404).end());
+
+
+/* Dataset and narrative paths hit Auspice's entrypoint.
+ */
+const coreBuilds = [
+  "/dengue",
+  "/ebola",
+  "/enterovirus",
+  "/FFV",
+  "/flu",
+  "/lassa",
+  "/measles",
+  "/mers",
+  "/mumps",
+  "/seattleflu", // XXX TODO: Remove once PR #60 is merged.
+  "/tb",
+  "/WNV",
+  "/yellow-fever",
+  "/ZEBOV",
```

Comment:
~As far as i'm aware this isn't a nextstrain.org dataset. https://nextstrain.org/ZEBOV doesn't work (although why the client crashes rather than displaying a 404 is a different matter).~ This dataset shouldn't be on nextstrain-data & can be removed from this list.

Reply:
Done.